### PR TITLE
TELCODOCS-853 - Document Fixed Issues

### PR DESCRIPTION
**Pull request by slovern**
[TELCODOCS-853](https://issues.redhat.com//browse/TELCODOCS-853) - Document Fixed Issues

Version(s): 4.12

Issue: https://issues.redhat.com/browse/TELCODOCS-853

Link to docs preview:

https://54571--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-bug-fixes

-  BZ#2049306 in section **Scalability and performance section** 
- OCPBUGS-4362 in section **Node**

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Review by MarSik — COMMENTED** [review]
/lgtm for the kubelet hard eviction thresholds change.

**Comment by slovern** [comment]
@mrniranjan can you provide QA lgtm for OCPBUGS-4362

**Comment by slovern** [comment]
@yliu127 can you provide QA lgtm for BZ#2049306

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -1713,6 +1715,8 @@ sourceStrategy:
 
 * Latency measurement tools, also known as `oslat`, `cyclictest`, and `hwlatdetect`, now run on completely isolated CPUs without the helper process running in the background that might cause latency spikes, thus providing more accurate latency measurements. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-2618[*OCPBUGS-2618*].
 
+* Previously, although the reference `PolicyGenTemplate` for `group-du-sno-ranGen.yaml` includes two `StorageClass`  entries, the generated policy included only one. With this update, the generated policy now includes both policies. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2049306[*BZ#2049306*].
```
Comment (yliu127) [inline release_notes/ocp-4-12-release-notes.adoc:1718]: /lgtm

File: release_notes/ocp-4-12-release-notes.adoc [hunk 2]
```diff
@@ -1611,6 +1611,8 @@ sourceStrategy:
 
 * Previously, a symlinks error message was printed out as raw data instead of formatted as an error, making it difficult to understand. This fix formats the error message properly, so that it is easily understood. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977660[*BZ#1977660*])
 
+* Previously, kubelet hard eviction thresholds were different from `k8s` defaults when a performance profile was applied to a node. With this release, the defaults have been updated to match the expected kubernetes defaults. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-4362[*OCPBUGS-4362*].
```
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:1614]:
```suggestion
* Previously, kubelet hard eviction thresholds were different from Kubernetes defaults when a performance profile was applied to a node. With this release, the defaults have been updated to match the expected Kubernetes defaults. (link:https://issues.redhat.com/browse/OCPBUGS-4362[*OCPBUGS-4362*]).
```

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [inline release_notes/ocp-4-12-release-notes.adoc:1718]:
```suggestion
* Previously, although the reference `PolicyGenTemplate` for `group-du-sno-ranGen.yaml` includes two `StorageClass`  entries, the generated policy included only one. With this update, the generated policy now includes both policies. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049306[*BZ#2049306*])
```

**Review by jeana-redhat — COMMENTED** [review]
Should match standard rel notes formatting as shown here. Also needs Kubernetes name fix.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 2 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:1614]: thanks, corrected

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (slovern) [reply release_notes/ocp-4-12-release-notes.adoc:1718]: sorry, same mistake twice. corrected now

**Pull request merged**